protocol/src/lib.rs

```diff
@@ -5,5 +5,10 @@ pub mod authentication;
 pub mod keyexchange;
 pub mod mercury;
 pub mod metadata;
+pub mod playlist4changes;
+pub mod playlist4content;
```

Comment:
Do you need to export all of these? If not, is there any downside in doing so i.e. crate size?

Reply:
Hmm no probably not, I think only playlist4changes is needed. Will update!

Reply:
Only playlist4changes need "pub", I'll remove it on the others.